### ISSUE TITLE
[Bug] progress chart has additional rect

### ISSUE THREAD

**Issue by skie1997** [VisActor/VChart#1611]
### Version

1.6.3

### Link to Minimal Reproduction

vscreen

### Steps to Reproduce

```ts
const spec = {
    "type": "circularProgress",
    "valueField": "current",
    "radiusField": "current",
    "padding": {
        "left": 6,
        "right": 6,
        "top": 6,
        "bottom": 6
    },
    "outerRadius": 0.7,
    "innerRadius": 0.5599999999999999,
    "cornerRadius": 20,
    "track": {
        "style": {
            "fill": "rgba(67,67,67,0.29)"
        }
    },
    "categoryField": "name",
    "roundCap": false,
    "data": [
        {
            "id": "data",
            "values": [
                {
                    "current": 0.2,
                    "name": "Profit",
                    "emptyName": "",
                    "progress": 0.2,
                    "detail": "",
                    "goalProgress": 1
                }
            ]
        }
    ],
    "axes": [
        {
            "type": "linear",
            "orient": "angle",
            "min": 0,
            "max": 1,
            "grid": {
                "visible": false
            },
            "label": {
                "visible": false
            },
            "ticks": false,
            "rangeType": "rough"
        }
    ],
    "indicator": {
        "visible": true,
        "title": {
            "visible": true,
            "autoLimit": true,
            "space": 10,
            "style": {
                "text": "Profit",
                "fontSize": 14,
                "fill": "#FFF",
                "fontWeight": "normal",
                "fontFamily": "D-DIN"
            }
        },
        "content": {
            "visible": true,
            "autoLimit": true,
            "style": {
                "text": "0.2000",
                "fontSize": 32,
                "fill": "#FFF",
                "fontWeight": "normal",
                "fontFamily": "D-DIN"
            }
        },
        "titleStyle": {
            "fontWeight": "normal"
        },
        "valueStyle": {
            "fontWeight": "normal"
        },
        "fixed": true,
        "gap": 10,
        "titleField": "name",
        "valueField": "current"
    },
    "extensionMark": [
        {
            "type": "text",
            "dataId": "data",
            "visible": false,
            "style": {
                "text": "",
                "fontWeight": 500,
                "fontSize": 16,
                "fill": "#666666",
                "textAlign": "center",
                "textBaseline": "top",
                "dy": 12
            }
        }
    ],
    "region": [
        {
            "clip": true
        }
    ],
    "background": "rgba(0, 0, 0, 1)",
    "animation": true,
    "tooltip": {
        "style": {
            "panel": {
                "padding": {
                    "top": 5,
                    "bottom": 10,
                    "left": 10,
                    "right": 10
                },
                "backgroundColor": "rgba(47, 59, 82, 0.9)",
                "border": {
                    "color": "#fff",
                    "width": 0,
                    "radius": 4
                },
                "shadow": {
                    "x": 0,
                    "y": 4,
                    "blur": 12,
                    "spread": 0,
                    "color": "rgba(0, 0, 0, 0.2)"
                }
            },
            "titleLabel": {
                "fontSize": 14,
                "fontColor": "#fff",
                "align": "left",
                "lineHeight": 18
            },
            "keyLabel": {
                "fontSize": 12,
                "fontColor": "#fff",
                "align": "left",
                "lineHeight": 18
            },
            "valueLabel": {
                "fontSize": 12,
                "fontColor": "#fff",
                "align": "right",
                "lineHeight": 18
            },
            "shape": {
                "size": 10,
                "spacing": 10
            },
            "spaceRow": 10
        }
    },
    "crosshair": {
        "xField": {
            "line": {
                "style": {
                    "fill": "rgba(47, 59, 82, 0.9)"
                }
            }
        },
        "yField": {
            "line": {
                "style": {
                    "fill": "rgba(47, 59, 82, 0.9)"
                }
            }
        }
    },
    "plotLayout": {
        "clip": false
    },
    "progress": {
        "style": {
            "innerPadding": 10,
            "outerPadding": 10,
            "cornerRadius": 20
        }
    },
    "animationAppear": {
        "progress": {
            "oneByOne": true,
            "duration": 1000,
            "delay": 2000,
            "loop": true
        }
    },
    "animationUpdate": {
        "progress": {
            "loop": false
        }
    },
    "animationEnter": {
        "progress": {
            "loop": false
        }
    },
    "hash": "f69c8fb900c6507bc3878ccc2799314d",
    "width": 250,
    "height": 225
}

const vchart = new VChart(spec, { dom: CONTAINER_ID });
vchart.renderAsync();

// Just for the convenience of console debugging, DO NOT COPY!
window['vchart'] = vchart;
```

### Current Behavior

![image](https://github.com/VisActor/VChart/assets/40675330/bf9361be-70ee-4692-896b-314ad5a641a7)

### Expected Behavior

there is no additional rect

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by xile611** [comment]
Fixed in VRender 1.7.4 
https://github.com/VisActor/VRender/pull/785

**Comment by skie1997** [comment]
it has been verified in vscreen
